content/docs/release-notes/2305-release-notes.md

```diff
@@ -0,0 +1,191 @@
+* Added support for stereoscopic rendering (i.e VR support) via OpenXr within O3de. This support was added through the Vulkan backend and it has been extensively tested using Quest 2. As part of this effort we are able to support Link mode where the rendering is done on PC and can be viewed on a stereoscopic display or the app can be run on the device natively. A special optimized VR pipeline has been added to Atom to help support running the app natively on the device.
+* The texture streaming support has been expanded to Vulkan backend. Now we have image streaming control which can load or evict texture mips automatically at runtime based on the streaming image pool's budget.
+* The shader library, material system, and builders have been updated to support a layer of abstraction between lighting and materials, called the material pipeline. It uses new builders and new features from the asset system to build unique shaders for each render pipeline. Material types can provide custom data structures and evaluator functions for vertex, pixel, geometry, and lighting stages. The builder inserts any of the custom include files with overridden data types and functions as it builds all of the combinations of shaders listed in the material pipeline.  
+	* The result of this is a concrete material type referencing all of the different combinations of shaders for each pipeline, with the customization stitched in. All of this is saved to the intermediate assets folder and processed by the AP to generate Final product material types and shaders. The material pipeline also includes a Lua script that will be invoked whatever material properties or the active render pipeline changes to enable or disable shaders based on the current state.  
+	* All of the PBR as well as other shaders and material types included with atom have been converted to the system.  
+	* All of the ASV samples and tests have also been updated to use this system. ASV also includes different experimental render and material pipelines to demonstrate how existing materials that use this system can be applied to a standard, deferred rendering pipeline.
+	* Known issues:  
+		* It takes more time to compile shaders for all of the active pipelines.
+		* Material Editor and material canvas crash switching to Multiview at XR pipelines.
+		* The organization of all of the shader and material files needs to be simplified to make it easier to find shader and material files.
+		* There are several to do items as a result of these changes that need to be addressed.
+* The TAA pass has been split into two passes to allow for the copying of TAA's history buffer to be separated from the TAA algorithm itself. This required a change to PostProcessParent.pass to reference `TaaParentTemplate` instead of `TaaTemplate`. If your project overrides PostProcessParent.pass and uses TAA, then it will need to be updated to point to the new TAA parent pass.
+
+
+## sig-network
```

Comment:
Please pull in the revised sig-network notes from https://github.com/o3de/sig-release/blob/main/releases/23.05.0/23050%20feature%20list.md